rest/data/test_test.go

```diff
@@ -105,51 +103,51 @@ func TestGetTestCountByTaskIdAndFilter(t *testing.T) {
 
 	for i := 0; i < numTasks; i++ {
 		taskId := fmt.Sprintf("task_%d", i)
-		count, err := serviceContext.GetTestCountByTaskIdAndFilters(taskId, "", []string{}, 0)
+		count, err := task.GetTestCountByTaskIdAndFilters(taskId, "", []string{}, 0)
```

Comment:
All tests (like this one) that now no longer test functions in the rest/data package should be moved to the appropriate package. We can split that to a different ticket if you prefer.

Reply:
I took a crack at it and it looks like moving all the tests around caused quite a few context errors (and I've since reverted those changes) and might be slightly more work than anticipated. I created https://jira.mongodb.org/browse/EVG-16570 to bring in next sprint.